api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -24,6 +25,9 @@ Workspace findByWorkspaceNamespaceAndFirecloudNameAndActiveStatus(
   Workspace findByWorkspaceNamespaceAndNameAndActiveStatus(
       String workspaceNamespace, String name, short activeStatus);
 
+  @Query("SELECT distinct w.workspaceNamespace, w from Workspace w")
```

Comment:
nit: make DISTINCT and FROM all caps. Do we have a SQL linter?

Reply:
I couldn't get this query to work locally. Did you mean `workspace_namespace`. I don't think you need to make `w` an alias in this case. This worked for me
`SELECT DISTINCT workspace_namespace FROM workspace ORDER BY NULL`.

Reply:
It's not MySQL, which makes sense.